Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.11.11-slim
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` should be specified as an absolute path. Change `app/` to `/app` to ensure that the working directory is correctly set.